### PR TITLE
installer: add Converter submod support & touch it in other places too

### PR DESCRIPTION
**Pull request by escalonn**

### CODE REVIEW

**Comment by escalonn** [comment]
one thought: _maybe_ when installing HIP with `Converter == False`, it should check for what the Converter target folder _would_ be and remove it if present regardless. [current code just uses the `scaffoldMod` function for both base mod and Converter mod, which removes only if it's going to install that mod]

File: installer/main.py [hunk 1]
```diff
@@ -1089,17 +1113,17 @@ def main():
...
 
-#        if Converter:
-#            euModFilename = scaffoldMod(euFolderBase,
-#                                        euFolder,
-#                                        'HIP_Converter',
-#                                        'HIP Converter Support',
-#                                        euSubfolder,
-#                                        eu4Version='1.10')
+        if Converter:
+            euModFilename = scaffoldMod('.',
```
Comment (zijistark) [inline installer/main.py]: I'm guessing `euModFileName` is never referenced after this assignment?

**Comment by zijistark** [comment]
Sounds like a good idea. [I think?]

**Comment by zijistark** [comment]
Eh, yes, that's what I think. Good idea.

File: installer/main.py (hunk 1 above)
Comment (escalonn) [reply installer/main.py]: right

**Pull request merged**